scripts/zones/Port_Bastok/npcs/Hilda.lua

```diff
@@ -24,8 +111,6 @@ entity.onTrade = function(player, npc, trade)
 end
 
 entity.onTrigger = function(player, npc)
-    local wildcatBastok = player:getCharVar("WildcatBastok")
```

Comment:
This shouldn't be deleted? As this is being checked below:

utils.mask.getBit(wildcatBastok, 3)

Reply:
Was a merge mistake I missed. Got that fixed and made sure this isn't happening in any of the other files.